velox/docs/functions/spark/comparison.rst

```diff
@@ -10,18 +10,27 @@ Comparison Functions
 
 .. spark:function:: equalnullsafe(x, y) -> boolean
 
-    Returns true if x is equal to y. Supports all scalar types. The
-    types of x and y must be the same. Unlike :spark:func:`equalto` returns true if both inputs
-    are NULL and false if one of the inputs is NULL.
+    Returns true if ``x`` is equal to ``y``. Supports all types. The
```

Comment:
`Supports all types.` maybe inaccurate, we don't support UNKNOWN?

Reply:
Updated, thanks